server/models/signature.model.js

```diff
@@ -0,0 +1,21 @@
+const neode = require('../schema/index');
+
+let Signatures = {};
+
+Signatures.create = ({personId, certificationId, signaturePersonId}) => {
+  return Promise.all([
+    neode.first('Person', 'id', personId),
+    neode.first('Certification', 'id', certificationId)
+  ])
+  .then(([person, certification]) => {
+    return person.relateTo(certification, 'has_certification_signature', {
+      signature_person_id: signaturePersonId,
+      signature_date: (new Date()).getTime()
```

Comment:
I thought we were going to make this so the frontend could define the date signed. If none specified, default to the above.